SpatialGDK/Source/SpatialGDK/Private/Utils/RPCContainer.cpp

```diff
@@ -66,8 +148,23 @@ bool FRPCContainer::ObjectHasRPCsQueuedOfType(const Worker_EntityId& EntityId, E
 
 	return false;
 }
+ 
+void FRPCContainer::BindProcessingFunction(const FProcessRPCDelegate& Function)
+{
+	ProcessingFunction = Function;
+}
 
-bool FRPCContainer::ApplyFunction(const FProcessRPCDelegate& FunctionToApply, const FPendingRPCParams& Params)
+bool FRPCContainer::ApplyFunction(const FPendingRPCParams& Params)
 {
-	return FunctionToApply.Execute(Params);
+	check(ProcessingFunction.IsBound());
+	FRPCErrorInfo ErrorInfo = ProcessingFunction.Execute(Params);
+	if (ErrorInfo.Success())
+	{
+		return true;
+	}
+	else
+	{
+		LogRPCError(ErrorInfo, Params);
```

Comment:
Wrap this in `UE_BUILD_SHIPPING`
Or perhaps even `NO_LOGGING`

Reply:
Wrapped in UE_BUILD_SHIPPING